test-it/ExpectedResults/QueueRefNumNamed.vtr

```diff
@@ -3,7 +3,7 @@ GetQueueStatus orig ref q: maxsize=-1 numElem=4 elts=(1 2 3 4)
 GetQueueStatus named queue q2 (clone ref): maxsize=-1 numElem=4 elts=(1 2 3 4)
 FlushQueue q (orig ref) remainingElts=(1 2 3 4)
 GetQueueStatus named queue q (orig ref) after clone ref q2 release: maxsize=-1 numElem=1 elts=(5) err=(false 0 '')
-GetQueueStatus named queue q2 after release: maxsize=-1 numElem=0 elts=() err=(true 1 'GetQueueStatus in TestProgram')
+GetQueueStatus named queue q2 after release: maxsize=0 numElem=0 elts=() err=(true 1 'GetQueueStatus in TestProgram')
 GetQueueStatus named queue q2 (reobtain clone ref): maxsize=-1 numElem=1 elts=(5) err=(false 0 '')
 GetQueueStatus named queue q2 (after orig disposed): maxsize=-1 numElem=1 elts=(5) err=(false 0 '')
-GetQueueStatus named queue q2 (after clone disposed): maxsize=-1 numElem=0 elts=() err=(true 1 'GetQueueStatus in TestProgram')
+GetQueueStatus named queue q2 (after clone disposed): maxsize=0 numElem=0 elts=() err=(true 1 'GetQueueStatus in TestProgram')
```

Comment:
Would be good to see the following test cases :
1. Calling GetQueueStatus on queue ref constant on which ObtainQueue is not called.
2. Calling GetQueueStatus on a unnamed queue ref after ObtainQueue and ReleaseQueue was called
3. Calling GetQueueStatus on a named queue ref after ObtainQueue and ReleaseQueue was called (which is covered in QueueRefNumNamed.vtr)
Ok to cover these in the VI test too.